frontend/app/src/people/widgetViews/BountyHeader.tsx

```diff
@@ -329,6 +329,36 @@ const BountyHeader = ({
     getPeopleList();
   }, [main, selectedWidget]);
 
+  const [counts, setCounts] = useState({
```

Comment:
it will be better to make the call once rather than making 3 calls to get the count for different statuses, I will make a new endpoint for this.

Reply:
Hmm.Okay.What changes should I make from my end

Reply:
Although it is already using Promise.all effectively to make asynchronous calls in parallel

Reply:
@VividhPandey003 use this endpoint `/filter/count`

Reply:
@VividhPandey003 don't make the calls too complex, take away the promise.all and just use this `/filter/count`, this endpoint gives the accurate count of the filter statuses, that was what it was made for.

Reply:
I have already removed promise.all. Instead I am using a single call to gobounties/filter/count. 

Attached image is the final code

![image](https://github.com/stakwork/sphinx-tribes/assets/91251535/dbe23845-c239-4825-8f81-ae9f72edff3e)